src/config/configAggregator.ts

```diff
@@ -408,3 +406,47 @@ export namespace ConfigAggregator {
     ENVIRONMENT = 'Environment',
   }
 }
+
+export class SfdxConfigAggregator extends ConfigAggregator {
+  public getPropertyMeta(key: string): ConfigPropertyMeta {
+    const match = this.getAllowedProperties().find((element) => key === element.key);
+    if (match?.deprecated && match?.newKey) {
```

Comment:
do we need all of this deprecation logic, if all `sfdx` config variables aren't deprecated?